src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -880,5 +919,63 @@ winrt::fire_and_forget IslandWindow::SummonWindow()
     LOG_IF_WIN32_BOOL_FALSE(ShowWindow(_window.get(), SW_SHOW));
 }
 
+bool IslandWindow::IsQuakeWindow() const noexcept
+{
+    return _isQuakeWindow;
+}
+
+void IslandWindow::IsQuakeWindow(bool isQuakeWindow) noexcept
+{
+    if (_isQuakeWindow != isQuakeWindow)
+    {
+        _isQuakeWindow = isQuakeWindow;
+        if (IsQuakeWindow())
+        {
+            _enterQuakeMode();
+        }
+    }
+}
+
+void IslandWindow::_enterQuakeMode()

```

Comment:
I don't see any monarch changes here... is something else stopping two windows from getting named `_quake`? You did say that someone could end up naming it that themselves and it would get the behavior reserved here... but I would think that ending up with multiple of them is bad. Also no idea how you'll block that cross-elevation...

Reply:
1. The window naming PR already prevented windows from having the same name
2. there's nothing that we can do to prevent having simultaneously an elevated and unelevated window with the same name. So you can have an Admin: \_quake and an unelevated one. It's a shame, but packaged COM & mixed elevation really don't play well 😢